tools/cpp/BUILD.windows.tpl

```diff
@@ -179,6 +180,7 @@ cc_toolchain_config(
         "objdump": "wrapper/bin/msvc_nop.bat",
         "strip": "wrapper/bin/msvc_nop.bat",
     },
+    default_link_flags = ["/MACHINE:X64"],
```

Comment:
This is not a bug fix, I added this new attribute `default_link_flags`, because `clang-cl` requires `/DEFAULTLIB:clang_rt.builtins-x86_64.lib` as well. And for later toolchain for arm and x86, this would also change to `/MACHINE:x86` and `/MACHINE:ARM`